compass/landice/tests/ismip6_forcing/ocean_thermal/process_thermal_forcing.py

```diff
@@ -0,0 +1,308 @@
+import os
+import shutil
+import subprocess
+import xarray as xr
+from compass.landice.tests.ismip6_forcing.create_mapfile \
+    import build_mapping_file
+from mpas_tools.io import write_netcdf
+from compass.step import Step
+
+
+class ProcessThermalForcing(Step):
+    """
+    A step for creating a mesh and initial condition for dome test cases
+    """
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.landice.tests.ismip6_forcing.ocean_thermal.
+                    OceanThermal
+            The test case this step belongs to
+        """
+        super().__init__(test_case=test_case, name='process_thermal_forcing',
+                         cores = 4, min_cores = 1)
```

Comment:
```suggestion
                         cores=4, min_cores=1)
```
For keyword arguments like these, no spaces around the equals sign.